runtime/autoload/zip.vim

```diff
@@ -395,8 +395,8 @@ fun! zip#Extract()
   endif
 
   " extract the file mentioned under the cursor
-"  call Decho("system(".g:zip_extractcmd." ".shellescape(b:zipfile)." ".shellescape(shell).")")
-  call system(g:zip_extractcmd." ".shellescape(b:zipfile)." ".shellescape(shell))
+"  call Decho("system(".g:zip_extractcmd." ".shellescape(b:zipfile)." ".shellescape(fname).")")
+  call system($"{g:zip_extractcmd} {shellescape(b:zipfile)} {shellescape(fname)}")
```

Comment:
The support for string interpolation is fairly recent
`v8.2.4883` (May 2022).  Substituting `fname` should be enough?

Reply:
I thought the same, it is more readable to my eye with the string interpolation